dist/stat_data.py

```diff
@@ -513,6 +513,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_pinned_snapshot_range', 'transaction range of IDs currently pinned by named snapshots', 'no_clear,no_scale'),
     TxnStat('txn_pinned_timestamp', 'transaction range of timestamps currently pinned', 'no_clear,no_scale'),
     TxnStat('txn_pinned_timestamp_oldest', 'transaction range of timestamps pinned by the oldest timestamp', 'no_clear,no_scale'),
+    TxnStat('txn_prepare', 'transactions prepared'),
```

Comment:
I think an additional `cache_read_deleted_prepared` statistic would be interesting. i.e: in `__wt_delete_page_instantiate`